### PR TITLE
docs: Additional documentation for existing functions

### PR DESCRIPTION
**Pull request by ernestojeda**
Please point out any punctuation issues. I will attempt to fix in this PR.

Signed-off-by: Ernesto Ojeda <ernesto.ojeda@intel.com>

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/edgexfoundry/ci-management/blob/main/.github/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Added labels
## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation content changes
- [ ] Other... Please describe:

## Sandbox Testing
Test Links :

### CODE REVIEW

**Review by soda480 — DISMISSED** [review]
Looks great.

File: vars/edgeXSnyk.groovy [hunk 1]
```diff
@@ -14,6 +14,61 @@
...
+
+ ```groovy
+ edgeXSnyk(
+    command: 'test',
+    dockerImage: 'nexus3.edgexfoundry.org:10004/core-command:latest',
+    dockerFile: '<path to Dockerfile>'
+ )
+ ```
+
+ Test docker image for vulnerabilities and send email of findings
```
Comment (bill-mahoney) [inline vars/edgeXSnyk.groovy:57]: "." missing

File: vars/edgeXSwaggerPublish.groovy [hunk 2]
```diff
@@ -14,14 +14,42 @@
...
-   Usage: edgeXSwaggerPublish(apiFolders: 'openapi/v1 openapi/v2') Defaults to owner:EdgeXFoundry1
-   Usage: edgeXSwaggerPublish(apiFolders: 'openapi/v1') Defaults to owner:EdgeXFoundry1
-   Usage: edgeXSwaggerPublish(owner: 'customOwner', apiFolders:'openapi/v1')
-   Usage: edgeXSwaggerPublish(owner: 'customOwner', apiFolders:'openapi/v1 openapi/v2')
+/**
+ # edgeXSwaggerPublish
+
+ ## Overview
+
+ Shared library containing to publish Swagger API docs up to [https://api.swaggerhub.com](https://api.swaggerhub.com)
```
Comment (bill-mahoney) [inline vars/edgeXSwaggerPublish.groovy:22]: Doesn't read right? Maybe "Shared library containing methods to publish ... " ? or just "Shared library to publish ... "?

File: vars/edgeXSwaggerPublish.groovy [hunk 3]
```diff
@@ -14,14 +14,42 @@
...
+
+ **NOTE:** `${APIKEY}` needs to be a pointer to a file with the key. This will need to be set locally from your environment or from Jenkins
```
Comment (bill-mahoney) [inline vars/edgeXSwaggerPublish.groovy:24]: Missing "."

**Review by bill-mahoney — APPROVED** [review]
Minor punctuation things, but LGTM

File: vars/edgeXSwaggerPublish.groovy (hunk 2 above)
Comment (ernestojeda) [reply vars/edgeXSwaggerPublish.groovy:22]: Nice catch

**Review by soda480 — APPROVED** [review]
LGTM

**Review by bill-mahoney — APPROVED** [review]
LGTM

**Pull request merged**